app/views/public_pages/stimulus.html.erb

```diff
@@ -56,7 +66,7 @@
       <div class="grid__item width-seven-twelfths">
         <h2 class="h3"><%= t("views.public_pages.stimulus.heading") %></h2>
         <ul class="list--bulleted">
-          <% t("views.public_pages.stimulus.facts_html").each do |item| %>
+          <% t("views.public_pages.stimulus.facts_html", :file_with_help_link => file_with_help_questions_path).each do |item| %>
```

Comment:
Found out that the link for this translation was not working for either English or Spanish. I changed it to pass the link in through a named variable.